Source/JavaScriptCore/assembler/AssemblerBuffer.h

```diff
@@ -439,6 +439,7 @@ namespace JSC {
 #if !CPU(ARM64) // If we were to define this on arm64e, we'd need a way to update the hash as we write directly into the buffer.
         void* data() const { return m_storage.buffer(); }
 #endif
+        const void* constData() const { return m_storage.buffer(); }
```

Comment:
This is ugly but unfortunately the `void* data() const` above means we can't also have a `const void* data() const`.